.github/workflows/bridge-ui.yml

```diff
@@ -0,0 +1,36 @@
+name: Bridge UI CI/CD
+
+on:
+  push:
+    branches-ignore:
+      - release-please-*
+    paths:
+        - "packages/bridge-ui/**"
+
+jobs:
+  deploy-bridge-ui-preview: 
+    if: ${{ !github.ref_name == 'main' }}  
+    uses: ./.github/workflows/bridge-ui-deploy.yml  
+    with:
+      environment: 'preview'
+      flags: ''
+      vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+      vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID_BRIDGE_UI }}
+
+  deploy-bridge-ui-preview-internal: 
+    if: ${{ !github.ref_name == 'main' }}
+    uses: ./.github/workflows/bridge-ui-deploy.yml
+    with:
+      environment: 'preview'
+      flags: ''
+      vercel_org_id: ${{ secrets.VERCEL_ORG_ID }}
+      vercel_project_id: ${{ secrets.VERCEL_PROJECT_ID_BRIDGE_UI_INTERNAL }}
+
+  deploy-bridge-ui-production: 
```

Comment:
We have at least 2 production deployment, one for mainnet, one for testnet. How about we have `deploy-bridge-ui-mainnet-production`, `deploy-bridge-ui-hekla-production`,`deploy-bridge-ui-mainnet-preview`, `deploy-bridge-ui-hekla-preview`